04-cond.Rmd

```diff
@@ -211,6 +211,23 @@ In this case, "either" means "either or both", not "either one or the other but
 > plot_dist(dat[, 10], threshold = 10, use_boxplot = FALSE)  # day (column) 10 - create histogram
 > plot_dist(dat[1:5, 10], threshold = 10)                    # samples (rows) 1-5 on day (column) 10
 > ```
+>   + As a further challenge, fill in the blanks of the function hist_max() below to create a function that does the following:  
```

Comment:
Since your challenge is unrelated to the previous challenge, please make yours a new challenge with its own title.